README.md

```diff
@@ -22,64 +22,81 @@ yarn add cdk-scheduler
+### Understand how the Scheduler works
 
-![architecture: dynamoDB with scheduled event / lambda scheduled every 15 minutes / publishes to SQS with delay](./docs/images/Architecture%20Scheduler.jpg)
+`cdk-scheduler` is powered by SQS feature to delay events up to 15 minutes.
+
+A lambda is scheduled to query a DynamoDB Table every 15 minutes, it pushes every events scheduled in the next 15 minutes to SQS with a delay corresponding the desired publication date.
 
-### Usage example with CDK - Typescript
+### Import and initialize Scheduler
 
-You can check out the full implementation example in [app.ts](./bin/app.ts).
+Import the scheduler with :
 
 ```ts
 import { Scheduler } from 'cdk-scheduler';
+```
+
+Then instantiate the scheduler :
 
-class AppStack extends Stack {
-  constructor(scope: Construct, id: string) {
-    super(scope, id);
-
-    // Create Scheduler Construct
-    const schedulerLib = new Scheduler(this, 'scheduler-lib');
-
-    // Grant writing role to Dynamo DB to your service that will
-    const dynamoDbApiIntegrationRole = new Role(
-      this,
-      'DynamoDbApiIntegrationRole',
-      { assumedBy: new ServicePrincipal('apigateway.amazonaws.com') },
-    );
-
-    schedulerLib.schedulerTable.grantWriteData(dynamoDbApiIntegrationRole);
-
-    // Add an integration tool such as Dynamo integration or Lambda
-    const integration = new DynamoDBPutItemIntegration({
-      partitionKey: schedulerLib.partitionKeyValue,
-      table: schedulerLib.schedulerTable,
-      role: dynamoDbApiIntegrationRole,
-    });
-
-    const restApi = new RestApi(this, 'RestApi');
-
-    const addScheduledEventModel = restApi.addModel('AddScheduledEventModel', {
-        ...
-    });
-
-    restApi.root.addMethod('POST', integration, {
-      methodResponses: [{ statusCode: '200' }],
-      requestModels: {
-        'application/json': addScheduledEventModel,
-      },
-      requestValidatorOptions: {
-        validateRequestBody: true,
-        validateRequestParameters: true,
-      },
-      // authorizer: new RequestAuthorizer(stack, 'MyAuthorizer', {}),
-    });
-  }
-}
-
-const app = new App();
-new AppStack(app, 'AppStack');
+```ts
+const myScheduler = new Scheduler(app, id);
 ```
 
+- `app` : Your CDK app (a `Construct`)
+- `id` : The id/name of your scheduler (a `string`)
+
+### Write messages to the scheduler
+
+Grant access to write a new message to the scheduler to the service(s) that will write to it. For example an API Integration or a lambda function :
+
+```ts
+myScheduler.schedulerTable.grantWriteData(newMessageLambda);
+```
+
+Then you can post new messages to the scheduler by inset a new row into the `myScheduler.schedulerTable` dynamoDB table. The parameters you need to do so are the following :
```

Comment:
Then, you can post new messages to the scheduler by inserting a new row into the `myScheduler.schedulerTable` DynamoDB table.
Parameters needed are:

Reply:
@guiyom-e  Done !